### DISCUSSION TITLE
run impoters and api on different machine

### DISCUSSION THREAD

**Discussion by MALKARAJ** [osm-search/Nominatim#2806] [Q&A]
Hi Team,
is it possible to run impoters and API on different machine, kindly give me some hints about this

**Comment by mtmail** [comment]
Related or duplicate of https://github.com/mediagis/nominatim-docker/issues/358 The docker approach can work: import on one machine, [export the docker container ](https://docs.docker.com/engine/reference/commandline/export/) into a tar file. Copy the file to another, for example less powerful, machine and run the API. Setups where multiple machines share the same database (reading, writing) are complex and error-prone.

**Reply by MALKARAJ** [reply]
Hi Team,
Thanks for your valuable feedback,
Kindly Throw some light on how to separate and what the important configuration needed to make to achieve this

**Reply by mtmail** [reply]
I believe https://github.com/mediagis/nominatim-docker/issues/245 covers that. You already posted there but didn't reply when a maintainer suggested a fix.
